remote-monitoring/single-vm-setup.sh

```diff
@@ -35,6 +35,7 @@ export PCS_AUTH_AUDIENCE="$6"
 export PCS_WEBUI_AUTH_TYPE="aad"
 export PCS_WEBUI_AUTH_AAD_TENANT="$5"
 export PCS_WEBUI_AUTH_AAD_APPID="$6"
+export PCS_APPLICATION_SECRET=$(cat /dev/urandom | LC_CTYPE=C tr -dc 'a-zA-Z0-9-,./;:[]\(\)_=^!~' | fold -w 64 | head -n 1)
```

Comment:
Will need to add this for k8s as well. Are you just generating a random guid?

Reply:
it's actually a random string of 64 chars